controllers/metal3machine_controller.go

```diff
@@ -115,20 +114,7 @@ func (r *Metal3MachineReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	// Fetch the Cluster.
 	cluster, err := util.GetClusterFromMetadata(ctx, r.Client, capiMachine.ObjectMeta)
 	if err != nil {
-		machineLog.Info("Metal3Machine's owner Machine is missing cluster label or cluster does not exist")
-		setErrorM3Machine(capm3Machine,
-			"Metal3Machine's owner Machine is missing cluster label or cluster does not exist",
-			capierrors.InvalidConfigurationMachineError,
-		)
-
-		return ctrl.Result{}, errors.Wrapf(err, "Metal3Machine's owner Machine is missing label or the cluster does not exist")
-	}
-	if cluster == nil {
-		setErrorM3Machine(capm3Machine, fmt.Sprintf(
-			"The machine is NOT associated with a cluster using the label %s: <name of cluster>",
-			capi.ClusterLabelName,
-		), capierrors.InvalidConfigurationMachineError)
-		machineLog.Info(fmt.Sprintf("Please associate this machine with a cluster using the label %s: <name of cluster>", capi.ClusterLabelName))
+		machineLog.Info("Machine is missing cluster label or cluster does not exist")
 		return ctrl.Result{}, nil
```

Comment:
shall we use this now instead return `ctrl.Result{Requeue: true, RequeueAfter: requeueAfter}, nil`

Reply:
Sorry if I didn't understand. Are you suggesting to add re-queuing here?

Reply:
yes.

Reply:
Perhaps no, after https://github.com/metal3-io/cluster-api-provider-metal3/pull/454#discussion_r777348701 because we are not setting an actual error.

Reply:
would it be helpful to re-add the logging part ?
We keep the log but ignore the result as is done now so that we visibility towards the logic.
https://github.com/metal3-io/cluster-api-provider-metal3/pull/454/files#diff-250f2503d81624c7304619762d56ee502ff1462e5ad0c965af1eb9bfed5f340cL118

Reply:
We still have it, it is moved to https://github.com/metal3-io/cluster-api-provider-metal3/pull/454/files#diff-250f2503d81624c7304619762d56ee502ff1462e5ad0c965af1eb9bfed5f340cR119